experiment/project.clj

```diff
@@ -3,6 +3,7 @@
   :dependencies [[org.clojure/clojure "1.8.0"]
                  [thinktopic/cortex "0.9.9-SNAPSHOT"]
                  [thinktopic/think.image "0.4.10"]
+                 [org.shark8me/tfevent-sink "0.1.2-SNAPSHOT"]
```

Comment:
Can you make a non-snapshot version of this library?

We prefer not to depend on 3rd party snapshots.